src/node_errors.cc

```diff
@@ -978,6 +978,9 @@ void TriggerUncaughtException(Isolate* isolate,
 
   // Now we are certain that the exception is fatal.
   ReportFatalException(env, error, message, EnhanceFatalException::kEnhance);
+#if HAVE_INSPECTOR
+  profiler::EndStartedProfilers(env);
```

Comment:
Uh just realized we should call `profiler::EndStartedProfilers` instead since the whole `WaitForInspectorDisconnect` resets signals.